services/integration/src/models/constitution/PreviousInformation.model.js

```diff
@@ -0,0 +1,30 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const previousInformationSchema = new Schema({
+  description: {
+    type: String,
+    required: [true, ' Description is obligatory field']
+  },
+  vision: {
+    type: String,
+    required: [true, 'Vision obligatory field']
+  },
+  viability: {
+    type: String,
+    required: [true, 'Viability obligatory field']
+  },
+  requirements: {
+    type: String,
+    required: [true, 'Requirements obligatory field']
+  },
+  client: {
```

Comment:
this field must be an oibject with this info: 
![image](https://user-images.githubusercontent.com/47579072/58766327-cd499c80-8542-11e9-8f51-3c8486f22444.png)

Reply:
ok , but in trello you dont say this